misk-aws-dynamodb/build.gradle

```diff
@@ -0,0 +1,15 @@
+dependencies {
+  implementation dep.guice
+  implementation dep.awsDynamodb
+  implementation project(':misk-aws')
```

Comment:
is this needed?

Reply:
Yes. The `AwsRegion` class is needed from that package.